packages/appium-tizen-tv-driver/lib/scripts.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * These are scripts which are sent to the browser via an `/execute/<async|sync>` command.
+ * @module
+ */
+
+/**
+ * All functions must have a final parameter which is a {@linkcode AsyncCallback}.
+ *
+ * This is not expressible dynamically in TS, so I didn't do it
+ */
+export const AsyncScripts = Object.freeze({
+  /**
+   * @param {number|string} code
+   * @param {string} key
+   * @param {number} duration
+   * @param {AsyncCallback} done
+   * @returns {void}
+   */
+  pressKey: (code, key, duration, done) => {
```

Comment:
so cool that this will just get stringified as an exec-able function 😲

Reply:
I remembered `Function.prototype.toString()` is a thing and wondered if it was kosher in this context, so I looked at the wdio source and they do the same thing.  if it breaks, blame bromann